extension/src/experiments/commands/register.ts

```diff
@@ -129,11 +129,7 @@ const registerExperimentInputCommands = (
 ): void => {
   internalCommands.registerExternalCliCommand(
     RegisteredCliCommands.EXPERIMENT_BRANCH,
-    () =>
-      experiments.getCwdExpNameAndInputThenRun(
-        getBranchExperimentCommand(experiments),
-        Title.ENTER_BRANCH_NAME
-      )
+    () => experiments.createExperimentBranch()
```

Comment:
[F] This function is no longer general so may as well give it a better name and remove the need to pass values.